demisto_sdk/commands/init/templates/HelloIAMWorld/HelloIAMWorld.py

```diff
@@ -40,10 +40,10 @@ def get_user(self, email: str) -> Optional[IAMUserAppData]:
         res = self._http_request(
             method='GET',
             url_suffix=uri,
-            params=query_params
+            params=query_params,
         )
 
-        if res and len(res.get('result', [])) == 1:  # TODO: make sure you verify a single result was retrieved
+        if res and res.get('result', []):  # TODO: make sure you verify a single result was retrieved
```

Comment:
according to the comment, there should be exactly "one" result, not "at least one".
in that case, the original code is more accurate